### ISSUE TITLE
Report of Alt text on CMS images

### ISSUE THREAD

**Issue by NWatson212** [visitscotland/dot-com-brxm#53]
Create a report that can be run on demand by content team to identify CMS images that contain Alt text in any language.

The decision to remove Alt text was made during accessibility assessment.  The external assessors advised that alt text on decorative images is not only 'not required' but would fail accessibility audit.

Ticket to remove alt text was VS-4408
